java/arcs/sdk/wasm/WasmTupleHandle.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.sdk.wasm
+
+/** Combined Handle to allow events on two handles to trigger actions. */
+class WasmTupleHandle<T, U>(
```

Comment:
Can you describe what else you think will make sense to put on this object?

I am wondering what is the upside of having this object at all vs. having just a method for registering updates, i.e.

combineUpdates(Handle<A>, Handle<B>, Handle<C>) {a, b, c ->
  (...)
}